pkg/restapi/issuer/operation/operations.go

```diff
@@ -395,6 +402,33 @@ func (o *Operation) storeCredentialHandler(rw http.ResponseWriter, req *http.Req
 	o.storeVC(data, vc, rw)
 }
 
+// Change Log Specification swagger:route PUT /logspec issuer changeLogSpecReq
+//
+// Changes the current log specification.
+// Format: ModuleName1=Level1:ModuleName2=Level2:ModuleNameN=LevelN:AllOtherModuleDefaultLevel
+// Valid log levels: critical,error,warn,info,debug
+//
+// Note that this will not work properly if a module name contains an '=' character.
+//
+// Responses:
+//    default: genericError
+//        200: emptyRes
+func logSpecPutHandler(rw http.ResponseWriter, req *http.Request) {
+	common.ChangeLogSpec(rw, req)
+}
+
+// Get Current Log Specification swagger:route GET /logspec issuer getLogSpecReq
+//
+// Gets the current log specification.
+// Format: ModuleName1=Level1:ModuleName2=Level2:ModuleNameN=LevelN:AllOtherModuleDefaultLevel
+//
+// Responses:
+//    default: emptyRes
+//        200: getLogSpecRes
+func (o *Operation) logSpecGetHandler(rw http.ResponseWriter, _ *http.Request) {
+	common.GetLogSpec(rw, o.stringBuilder())
+}
+
```

Comment:
Why do we have same code/handler in issuer and [verifier](https://github.com/trustbloc/edge-service/pull/459/files#diff-102a7d6898cbfe029d0c48e1c41fe8b6R349-R375) ?

Reply:
Oops - that's a mistake. After re-examining the start.go code I figured out the correct way to register the handler in a way that's common to the various services.

Fixed. I also added the handler to the DID service since I had missed adding it there.

Reply:
I still see code/models defined in [issuer](https://github.com/trustbloc/edge-service/pull/459/files#diff-2cdb1df56975c44c0663a77dfe24f0c1R149-R170) and [verifier](https://github.com/trustbloc/edge-service/pull/459/files#diff-c795f06537a744401dc78acce4e41b11R100) packages.

Can we try to rename pkg/restapi/common/common.go to pkg/restapi/log, combine get/put handlers and call from the start.go ? This will keep the logger handler/code in one package.

Reply:
Updated the code per our conversation earlier today